tests/Dockerfile.daemon

```
@@ -33,14 +33,6 @@ FROM ubuntu:22.04
 RUN mkdir -p /run/dbus
 RUN apt update && apt install -y openssh-server libboost-log1.74.0 liblmdb0 libarchive13 dbus iproute2
```

We can remove here the package too
```suggestion
RUN apt update && apt install -y libboost-log1.74.0 liblmdb0 libarchive13 dbus iproute2
```